tests/functional/i/invalid/invalid_bases_assignment.py

```diff
@@ -0,0 +1,16 @@
+# pylint: disable=missing-module-docstring, missing-class-docstring
+# pylint: disable=too-few-public-methods
+
+
+class Apple:
+    pass
+
+
+Apple.__bases__ = ()  # [invalid-bases-assignment]
+Apple.__bases__ = "lala"  # [invalid-bases-assignment]
+Apple.__bases__ = True  # [invalid-bases-assignment]
+Apple.__bases__ = (1,)
+Apple.__bases__ = ("red", "green")
+
+bases = ("orange", "yellow")
+Apple.__bases__ = bases  # [invalid-bases-assignment]
```

Comment:
Why is this invalid?

Reply:
HA as I wrote this I inverted what's allowed for this check in my mind! Of course this should've passed. Will take a look.

Reply:
ah so turns out this is correct, both for the assigning to `x` and straight without assignment. it's because `Apple.__bases__ = ("red", "green")` gets a `TypeError: Apple.__bases__ must be tuple of classes, not 'str'` in the REPL. Also mentioned in a comment above  for ints. I'll update the definition for this checker to be more strict.

Reply:
Oke, but:
```python
bases = (int, str)
Apple__bases__ = bases
```

That works right?

Not to discourage you, but this is shaping up to be a lot more difficult than we might have originally envisioned.

Reply:
Huh yeah that does work, didn't expect it. Let me summarize the findings and see how we can limit the scope here, if at all.